### PR TITLE
Dependi added

### PR DESCRIPTION
**Pull request by serayuzgur**

### CODE REVIEW

**Comment by serayuzgur** [comment]
@palfrey can you please help , it has a marketplace url for vscode but it fails

**Comment by palfrey** [comment]
I'd previously rejected this tool (see https://github.com/rust-unofficial/awesome-rust/pull/1762) as it's a Typescript tool that happens to have Rust support. Not of interest.

**Comment by serayuzgur** [comment]
> I'd previously rejected this tool (see #1762) as it's a Typescript tool that happens to have Rust support. Not of interest.

Hi, Dependi is a tool that helps with Cargo dependency management. It is written in TypeScript because VS Code supports that. If you look at the IDE section, you'll see that tools are written in different languages:

- Eclipse/IntelliJ = Java
- Sublime = Python, etc.

Also, every VS Code plugin is written in TypeScript or JavaScript. I don't understand your statement; can you please explain?

**Comment by palfrey** [comment]
Yes, that makes sense. Sorry, had a busy few weeks. You'll need to add the url to the list at https://github.com/rust-unofficial/awesome-rust/blob/main/src/main.rs#L47 to make this work.

**Comment by serayuzgur** [comment]
> Yes, that makes sense. Sorry, had a busy few weeks. You'll need to add the url to the list at https://github.com/rust-unofficial/awesome-rust/blob/main/src/main.rs#L47 to make this work.

Thanks, all done!

File: README.md [hunk 1]
```diff
@@ -801,6 +801,8 @@ See also [Are we (I)DE yet?](https://areweideyet.com/) and [Rust Tools](https://
     * [PistonDevelopers/VisualRust](https://github.com/PistonDevelopers/VisualRust) - A Visual Studio extension for Rust [![Build status](https://ci.appveyor.com/api/projects/status/5nw5no10jj0y4p3f?svg=true)](https://ci.appveyor.com/project/vosen/visualrust)
   * [Visual Studio Code](https://code.visualstudio.com/)
     * [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb) - A LLDB extension
+    * [Dependi](https://marketplace.visualstudio.com/items?itemName=fill-labs.dependi) - Manage your dependencies with ease
+
```
Comment (palfrey) [inline README.md:805]: Can you please remove this extra blank line?

**Pull request merged**